user/user_app.go

```diff
@@ -0,0 +1,68 @@
+package user
+
+type UserAppID string
+type AccountID string
+type AllowlistType string
+
+const (
+	AllowlistTypeContracts  AllowlistType = "contracts"
+	AllowlistTypeMethods    AllowlistType = "methods"
+	AllowlistTypeOrigins    AllowlistType = "origins"
+	AllowlistTypeServices   AllowlistType = "services"
+	AllowlistTypeUserAgents AllowlistType = "user_agents"
+)
+
+type UserApp struct {
+	ID                  UserAppID
+	AccountID           AccountID
+	PlanType            string
+	SecretKey           string
```

Comment:
1. What is this? #PUC
2. Should secrets be a separate struct?

Reply:
Comments have been updated.

Personally I would prefer not to create multiple structs. IMO the only purpose of the `UserApp` struct in PATH is to store settings like the secret key and so I think it makes semantic sense to have all fields at the base level. I prefer simplicity unless making things more complex has a clear benefit. Not strongly attached to this however.